glean-core/python/tests/metrics/test_timing_distribution.py

```diff
@@ -12,33 +12,28 @@
 from glean import _util
 
 
-def test_the_api_saves_to_its_storage_engine():
+def test_the_api_saves_to_its_storage_engine(monkeypatch):
```

Comment:
Not strictly necessary, but I wasn't aware of this feature in pytest until recently, and I think it improves some of the weirdness in the test.